conf/kubernetes-resource-staging-server.yaml

```diff
@@ -1,6 +1,6 @@
 ---
-apiVersion: extensions/v1beta1
-kind: ReplicaSet
+apiVersion: apps/v1beta1
+kind: Deployment
```

Comment:
Yes, Deployment from the group `extensions/v1beta1` will work on 1.5, 1.6 and 1.7, and we'll probably deprecate it in 1.8. So, I think we can use that for now, and not apps/v1beta1.